importanize/__main__.py

```diff
@@ -93,17 +93,21 @@ def from_path(cls, path):
             return cls.default()
 
     @classmethod
-    def find(cls, cwd=pathlib.Path.cwd(), root=None):
+    def find(cls, cwd=pathlib.Path.cwd(), root=None, config=None):
         path = cwd
+        config = config or cls.default()
 
-        while path != pathlib.Path(root or cwd.root):
```

Comment:
not sure when can this freeze here? this should go up the folder tree until root is encountered at which point loop should exit. what am I missing?

Reply:
Happens in cases when theres no root prefix for the specified file. for e.g 
`importanize utils/common.py`. 

`path` value lingers at `PosixPath('.')`

Reply:
ah, this method is called without resolving the path. oops